src/main/resources/templates/sge/command/qsub

```diff
@@ -38,7 +38,7 @@ qsub
 -o [(${logDir})]$JOB_ID.out
 [/]
 [# th:if="${envVariables != null}"]
--v "[(${envVariables})]"
+-v [(${envVariables})]
```

Comment:
and regarding this one, how a new algorithm will work with such command:

```
-v key=value;some_key=some value;some_key2-value2
```

If I am right it should split it into two parts, if so - we need to fix it, as I see there is two ways:
 - leave such template part as is (with `""`)
 - add for each env var quotes in case it has `spaces`

From my point of view, if it works, I would stick with first case (and also check that all other templates are fine regarding that)

Reply:
fixed by the first approach